instrumentation/java-http-client/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/javahttpclient/JavaHttpClientTest.java

```diff
@@ -22,37 +20,4 @@ public abstract class JavaHttpClientTest extends AbstractJavaHttpClientTest {
   protected HttpClient configureHttpClient(HttpClient httpClient) {
     return httpClient;
   }
-
-  @Nested
```

Comment:
I tested making these classes non-static, but then I hit circular dependency with the inner class extending its enclosing class

If we keep the classes static and remove the `@Nested` annotation, the tests run, but I believe that we risk test isolation problems